.github/workflows/ios_build_and_test.yml

```diff
@@ -2,22 +2,18 @@ name: ios_build_and_test
 
 on:
   push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
 
 jobs:
   build:
     strategy:
       matrix:
```

Comment:
Please ensure that you're using the same version of Xcode locally as you are on this action. Additionally, I recommend that you include the same versions of Xcode that partners use for their builds.

Reply:
+1.  I started trying to repro this locally and spent a while updating xcode.